runner/src/mill/runner/FileImportGraph.scala

```diff
@@ -33,7 +34,8 @@ case class FileImportGraph(
 object FileImportGraph {
   def backtickWrap(s: String): String = s match {
     case s"`$v`" => s
-    case _ => if (encode(s) == s) s else "`" + s + "`"
+    case _ => if (encode(s) == s) runtime.universe.asInstanceOf[runtime.JavaUniverse].quotedName(s)
```

Comment:
Rather than relying on `runtime.JavaUniverse`, let's instead port over the user-land implementation from Ammonite. That will make it a smoother upgrade to Scala 3 where `runtime.JavaUniverse` might not exist

https://github.com/com-lihaoyi/Ammonite/blob/b98bc05c0d90f43c46bafd69736ed22dd36f8734/amm/util/src/main/scala/ammonite/util/Model.scala#L127

Reply:
So I just grab the alphabet keywords from ammonite since NameTransformer should handle the other case and that exists in scala 3 stdlib